src/components/ContactInfo/ContactInfo.js

```diff
@@ -35,10 +35,11 @@ const ContactInfo = () => {
                         ></iframe>
                     </div>
                 </div>
-            </div>
 
-            <div className={cn.Copyright}>
-                © 2020 Junping Bergin PLLC, Incorporated. All Rights Reserved.
+                <div className={cn.Copyright}>
```

Comment:
I put Copyright back to where it was since it's too small to break it up - it'd be too fragmented otherwise! And the likelihood that we need code changes in this single <div> is very small.

Reply:
I put Copyright in a separate class because you said you wanted it to be visible in the Contact Us tab. Now it's only visible on the other pages and it isn't on the bottom of the page anymore. 
![ContactBar_0905](https://user-images.githubusercontent.com/34732378/92313467-d4026300-ef80-11ea-9b5f-f7b6d81c9b2f.PNG)
![ContactUs_page_0905](https://user-images.githubusercontent.com/34732378/92313468-d49af980-ef80-11ea-9c8c-e4c64a0226fc.PNG)